src/plugins/profiling/server/routes/load_topNStackTraces.ts

```diff
@@ -19,13 +19,15 @@ function transformFlamechart(src) {
   };
   Object.keys(src.TopNTraces).map((key) => {
     obj.TopN[key] = src.TopNTraces[key].map((item) => {
-      return { Value: item.TraceHash, Count: item.Count };
+      return { Value: item.StackTraceID, Count: item.Count };
```

Comment:
If the rename of `TraceHash` to `StackTraceID` is needed for changes to the Elasticsearch query, then I don't think we need to update here. The `load_*.ts` files load JSON fixtures originally captured from `pf-web-service`. Have we updated the web service responses?

Reply:
Good catch! I think for the sake of correctness, I should rename the relevant field in the fixtures... do you agree?

Reply:
I'd like to keep the original field name since the embedded Pixi flamegraph in this plugin depends on the same format returned by the web service. I also kept the fixtures in the same format as the web service so I could copy the responses from the web service APIs without modification.

Of course, if I'm missed any change in the web service's responses, we should definitely update the fixtures. :)

Reply:
Sounds reasonable to me.

Reply:
Ok thanks, will revert the `load*` ones